frontend/src/components/AppNavBar.vue

```diff
@@ -47,7 +47,9 @@ export default {
     justify-content: space-between;
 
     h3 {
+        font: var(--header-4);
         cursor: pointer;
+        font-weight: bold;
```

Comment:
This is the Login Signup text right? I think I prefer it not bold.. but meh

Reply:
Yeah I wasn't sure on that. I quite like the non-bold too.

Reply:
came about because of the change to use var(--header-3);